test/lib/ansible_test/_data/requirements/constraints.txt

```
@@ -13,5 +13,5 @@ pyparsing < 3.0.0 ; python_version < '3.5' # pyparsing 3 and later require pytho
 mock >= 2.0.0 # needed for features backported from Python 3.6 unittest.mock (assert_called, assert_called_once...)
 pytest-mock >= 1.4.0 # needed for mock_use_standalone_module pytest option
 setuptools < 45 ; python_version == '2.7' # setuptools 45 and later require python 3.5 or later
-pyspnego >= 0.1.6 ; python_version >= '3.10' # bug in older releases breaks on Python 3.10
+pyspnego >= 0.1.6 ; python_version >= '3.10' # older releases don't support current Python versions
```

This can be simplified now that Python 3.10 is the minimum:

```suggestion
pyspnego >= 0.1.6 # older releases don't support current Python versions
```

---

Just to clarify, the environment marker isn't needed since `pyspnego` is only used as a controller requirement.

---

Actually, now that I think about this more, it should be safe to drop this constraint entirely.

```suggestion
```